src/adaptation/ServiceGroups.cc

```diff
@@ -93,7 +93,7 @@ Adaptation::ServiceGroup::finalize()
             finalizeMsg("ERROR: Unknown adaptation name", serviceId, true);
         }
     }
-    debugs(93,7, HERE << "finalized " << kind << ": " << id);
+    debugs(93,7, "finalized " << kind << ": " << id);
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.